crates/jit/src/instantiate.rs

```diff
@@ -400,11 +402,9 @@ fn build_code_memory(
     ),
     String,
```

Comment:
While you're here, mind going ahead and changing this to an `anyhow::Result` to avoid using `String` as an error?